gammapy/datasets/tests/test_map.py

```diff
@@ -1052,15 +1052,15 @@ def test_stack_dataset_dataset_on_off():
     dataset_on_off.counts_off += 1
     dataset.stack(dataset_on_off)
 
-    assert_allclose(dataset.background_model.map.data, 0.2)
+    assert_allclose(dataset.background_model.map.data, 0.166667, rtol=1e-3)
 
 
 @requires_data()
 def test_info_dict_on_off(images):
     dataset = get_map_dataset_onoff(images)
     info_dict = dataset.info_dict()
     assert_allclose(info_dict["counts"], 4299, rtol=1e-3)
-    assert_allclose(info_dict["excess"], -22.52, rtol=1e-3)
+    assert_allclose(info_dict["excess"], -22.5135, rtol=1e-3)
```

Comment:
Again this should not have changed if no source model is defined on the `MapDatasetOnOff`.

Reply:
But `excess = counts - background`. `background` changed from `alpha*n_off` to `alpha/(alpha+1)*(counts+counts_off)`